plutus-tx/src/PlutusTx/AsData.hs

```diff
@@ -85,7 +85,13 @@ asDataFor dec = do
   -- The newtype declaration
   let ntD =
         let con = TH.NormalC cname [(TH.Bang TH.NoSourceUnpackedness TH.NoSourceStrictness, TH.ConT ''BuiltinData)]
-        in TH.NewtypeD [] name tTypeVars Nothing con derivs
+        in TH.NewtypeD [] name
+#if __GLASGOW_HASKELL__ >= 908
```

Comment:
could this be a condition on the version of the `template-haskell` package? that seems more direct

Reply:
Done.